README.md

```diff
@@ -1,4 +1,18 @@
 # Google Student Training in Engineering Program
 
 This repo contains Julia's portfolio and STEP projects.
-This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
\ No newline at end of file
+This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
+
+## Week 2-4 Projects: Portfolio
+
+View the completed website [here](http://juliasliu-step-2020.appspot.com/).
+
+To run the app locally, `cd` into the `portfolio` directory and `mvn package appengine:run`.
+
+## Week 5 Project: TDD (Test Driven Development)
+
+Code for the query() function is located in `step/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java`.
+
+JUnit tests are located under the `step/walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java`.
+
+To run the tests, `cd` into the `projects` directory and `mvn test` to run the JUnit tests.
```

Comment:
I think it should `project` not `projects`

Reply:
Yeah you're right, I'll change that and then try to merge.